CONTRIBUTING.md

```diff
@@ -3,11 +3,203 @@ Guidelines for Contributing
 
 Welcome to the D community and thanks for your interest in contributing! To get started, please read the [Starting as a Contributor](http://wiki.dlang.org/Starting_as_a_Contributor) article on the D Wiki.
 
-More Links
-----------
-
-* Fork [on Github](https://github.com/D-Programming-Language/phobos)
-* Use our [Bugzilla bug tracker](http://d.puremagic.com/issues/)
-* Follow the [Styleguide](http://dlang.org/dstyle.html)
-* Participate in [our forum](http://forum.dlang.org/)
-* Review Phobos additions in the [Review Queue](http://wiki.dlang.org/Review_Queue).
\ No newline at end of file
+Quick links
+-----------
+
+- Fork [on Github](https://github.com/D-Programming-Language/phobos)
+- Use our [Bugzilla bug tracker](http://d.puremagic.com/issues/)
+- Follow the [D style](http://dlang.org/dstyle.html)
+- Participate in [our forum](http://forum.dlang.org/)
+- Ask questions on `#d` IRC channel on freenode.org ([web interface](https://kiwiirc.com/client/irc.freenode.net/d))
+- Review Phobos additions in the [Review Queue](http://wiki.dlang.org/Review_Queue)
+
+How to make a good submission?
+------------------------------
+
+The usual workflow is:
+
+1. [Fork](https://help.github.com/articles/fork-a-repo/) [Phobos](https://github.com/D-Programming-Language/phobos) on Github
+2. Create your [own branch](https://github.com/Kunena/Kunena-Forum/wiki/Create-a-new-branch-with-git-and-manage-branches)
+3. Work locally on your new feature or fix (see the tips below)
```

Comment:
+1 done. However I kept the link to `tests` and link there to locally testing. The new `tests` section provides some helpful advice like using `rdmd -main -unittest` for single modules.